mocha-client-tests/README.md

```diff
@@ -0,0 +1,13 @@
+#Mocha client tests for WebExtensions
+##Install dependency
+`npm install` and `bower install`  
+
+And inside of addon please run `npm i`
+
+##Run with web-ext cli
+If you have it already run `npm run web-ext` if not `npm i web-ext -g` and then first command (will work with FF dev edition), if you have error with web-ext cli please add path for FF binary file with `--firefox-binary /path/to/firefox-bin`
```

Comment:
`npm run web-ext` doesn't work because `package.json` wants it to be `npm run web-ext-run`. I think you should rename the `script` in `package.json` to `web-ext`.